### PR TITLE
[EI-699] - Fix Range comments

### PR DESCRIPTION
**Pull request by StephanieBracaloni**

### CODE REVIEW

File: proto/iotics/api/common.proto [hunk 1]
```diff
@@ -36,7 +33,7 @@ message Offset {
 }
 
 // Range is the combination of the "Limit" and "Offset" is a request parameters. It is
-// used to return a specific range of results.
+// used to return a specific range of results. Default server values are applied.
```
Comment (vtermanis) [inline proto/iotics/api/common.proto]: Maybe:
> Default server values are applied if no limit is specified.

File: proto/iotics/api/common.proto [hunk 2]
```diff
@@ -20,11 +20,8 @@ option php_namespace = "Iotics\\Api";
 
 
 // Limit is a request parameter to limit the number of results.
-// The use of "Limit" is:
-// - if limit is not supplied or supplied with no limit value, assume default=500
-// - if limit is supplied assume max number of entries is whatever specified. The limit value is capped to 1000.
 message Limit {
-  // Max number of results
+  // Max number of results (under server max limit constraint)
```
Comment (vtermanis) [inline proto/iotics/api/common.proto]: I guess the issue here is that "server max limit" usually means "for the local host called" .. except for remote search - so maybe we should add a note to search regarding the limit.

How about e.g.:

> Maximum number of results (up to at most the space-configured maximum). Note that for remote requests the maximum depends on the remote, not local space.

(I'm not 100% happy about what I propose - but should we talk about servers or hosts or spaces & I think it's worth referencing the fact that not all limits are based on the local/configured one. Maybe a generic "Some requests have other limits which are documented therein")

File: proto/iotics/api/common.proto (hunk 1 above)
Comment (vtermanis) [reply proto/iotics/api/common.proto]: Also, as above - should we say server or host or space?

File: proto/iotics/api/twin.proto [hunk 3]
```diff
@@ -58,7 +58,7 @@ message Twin {
 // List all twins.
 message ListAllTwinsRequest {
   Headers headers = 1;
-  // Limit the results according to the value (optional: when not supplied, assume no default limits required - platform specific).
+  // Request range
```
Comment (vtermanis) [inline proto/iotics/api/twin.proto]: Search says: "Search request range"

.. would "Listing result range" be more applicable?

Comment (StephanieBracaloni) [reply proto/iotics/api/twin.proto]: https://github.com/Iotic-Labs/api/pull/10/commits/bc0f8fbc37582bd2a44de7b9caa16848389b4cda :+1:

File: proto/iotics/api/common.proto (hunk 1 above)
Comment (StephanieBracaloni) [reply proto/iotics/api/common.proto]: https://github.com/Iotic-Labs/api/pull/10/commits/bc0f8fbc37582bd2a44de7b9caa16848389b4cda 
Server or Host or Space => the 1 million pound question :grin: 
I removed the mention of server

File: proto/iotics/api/common.proto (hunk 2 above)
Comment (StephanieBracaloni) [reply proto/iotics/api/common.proto]: https://github.com/Iotic-Labs/api/pull/10/commits/bc0f8fbc37582bd2a44de7b9caa16848389b4cda 
I don't really want to expose this level of detail there.
The good practice if there is a range is to implement it especially if you don't have a full control on the default value

**Comment by StephanieBracaloni** [comment]
Thks @vtermanis

**Review by vtermanis — APPROVED** [review]
LGTM

**Pull request merged**